specs/commandline/inspect.md

```diff
@@ -0,0 +1,204 @@
+
+# notation inspect
+
+## Description
+
+Use `notation inspect` command to inspect all the signatures associated with signed artifact in a human readable format.
+
+Upon successful execution, both the digest of the signed artifact and the digests of signatures manifest along with their properties associated with signed artifact are printed in the following format:
+
+```shell
+<registry>/<repository>@<digest>
+├── application/vnd.cncf.notary.signature
+  ├──<digest_of_signature_manifest>
+    ├──<signed attributes>
+    ├──<user-defined attributes>
+    ├──<unsigned Attributes>
+    ├──<certificates>
+    ├──<payload>
+  ├──<digest_of_signature_manifest>
+    ├──<signed attributes>
+    ├──<unsigned attributes>
+    ├──<certificates>
+    ├──<payload>
+```
+
+## Outline
+
+```text
+Inspect all signatures with the signed artifact.
+
+Usage:
+    notation inspect [flags] <reference>
+    notation inspect <registry>/<repository>:<tag>
+
+Flags:
+   -h, --help for describing the signature
+   -o, --output on command line sets the output to json
+   -p, --password string password for registry operations (default to $NOTATION_PASSWORD if not specified)
+       --plain-http registry access via plain HTTP
+   -u, --username string username for registry operations (default to $NOTATION_USERNAME if not specified)
+```
+
+## Usage
+
+### Display the details of all the listed signatures and its associated certificate properties of the signed container image
+
+
+```text
+notation inspect <registry>/<repository>@<digest> [flags] <reference>
```

Comment:
Remove `<reference>` at the end since there is already `<registry>/<repository>@<digest>`. It's better to put `[flags]` before. An example:
```text
notation inspect [flags] <registry>/<repository>@<digest>
```